h2o-docs/src/product/data-science/algo-params/custom_distribution_func.rst

```diff
@@ -14,6 +14,7 @@ Use this option to specify a custom distribution function, which can then be use
 **Notes**: 
 
 - This option is only supported in the Python client.
+- Demos for the custom distribution and custom loss functions are located at https://github.com/h2oai/h2o-3/blob/master/h2o-py/demos/custom_loss_function_demo.ipynb.
```

Comment:
Change here and in the GBM chapter to:

"A demo for the custom distribution function is available here: https://github.com/h2oai/h2o-3/blob/master/h2o-py/demos/custom_loss_function_demo.ipynb"